axon-framework/events/event-processors/streaming.md

```diff
@@ -587,6 +587,75 @@ public class AxonConfig {
 {% endtab %}
 {% endtabs %}
 
+#### Token Stealing
+
+As described at the [start](#tracking-tokens), streaming processor threads can "steal" tokens from one another.
+You may regard a token as stolen when a thread loses a [token claim](#token-claims).
+Situations like this internally result in an `UnableToClaimTokenException`, caught by both streaming event processor implementations and translated into warn- or info-level log statements.
```

Comment:
```suggestion
Situations like this internally result in an `UnableToClaimTokenException,` caught by both streaming event processor implementations and translated into warn- or info-level log statements.
```
Does this mean there are 2 implementations?

Reply:
Yep! The Tracking Event Processor logs messages slightly differently from the Pooled Streaming Event Processor.